docs/workflow/README.md

```diff
@@ -16,10 +16,37 @@ The runtime repo can be built from a regular, non-admin command prompt. The repo
 
 For information about the different options available, supply the argument `-help|-h` when invoking the build script:
 ```
-libraries -h
+build -h
 ```
 On Unix, arguments can be passed in with a single `-` or double hyphen `--`.
 
+### Configurations
+
+Sometimes you want to build this repo in a mixture of configurations. First, a quick reminder of some concepts -- see the [glossary](../project/glossary.md) for more on these:
+
+* CoreCLR (often referred to as the runtime, most code under src/coreclr) -- this is the execution engine for managed code. It is written in C/C++. When built in a checked configuration, it is easier to debug into it, but it executes managed code more slowly - so slowly it will take a long time to run the managed code unit tests
```

Comment:
Checked configuration does not make debugging easier. Debugging checked configuration is about as hard as debugging release configuration.

The checked configuration have extra checks that catch hard to find bugs early. It is what makes it slow.

The debug configuration have these checks too, but it is also compiled with optimizations off so that it easy to debug. It is what makes it super slow.

Reply:
Oops, yes.